src/lib/server.ts

```diff
@@ -170,8 +172,11 @@ interface ZwavejsServerOptions {
 }
 
 export class ZwavejsServer {
+  // @ts-ignore
```

Comment:
What are we ignoring? Also, instead of ignore, we should use "expect error" so if anything ever changes, we are notified.

Reply:
This is temporary until I found how to fix the error typescript is showing